config.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
I don't know why this file isn't in the gitignore list. You shouldn't add this file due to it's overwritten on every `apply:xx` by `app-template/config-template.xml.

Reply:
That was my mistake! I just removed it.